src/components/global/Playground/index.tsx

```diff
@@ -444,11 +468,31 @@ export default function Playground({
     );
   }
 
+  function renderConsole() {
+    return (
+      <div className="playground__console">
+        <div className="playground__console-header">
+          <code>Console</code>
+        </div>
+        <div className="playground__console-body" ref={consoleBodyRef}>
+          {(ionicMode === Mode.iOS ? iosConsoleItems : mdConsoleItems).map((consoleItem, i) => (
```

Comment:
This isn't part of the design document, so free feel to push back and I can log a separate task to track this.

When the console UI is initially blank, it is a little unclear to the user what to expect. I think a placeholder message could be a nice improvement over this. Something simple like:
> Console messages will appear here when logged from the example above.

Reply:
I like that idea 👀 Will go ahead and add it since it should be simple.